packages/mui-joy/src/Input/Input.tsx

```diff
@@ -184,7 +181,7 @@ export const StyledInputHtml = styled('input')<{ ownerState: InputOwnerState }>(
   }),
 );
 
-export const StyledInputStartDecorator = styled('span')<{ ownerState: InputOwnerState }>(
+export const StyledInputStartDecorator = styled('div')<{ ownerState: InputOwnerState }>(
```

Comment:
We should update the `@default` tags in JSDoc here:
https://github.com/mui/material-ui/blob/5c729c744a0bb634bd7754a2328acfdf5cbf6078/packages/mui-joy/src/Input/InputProps.ts#L21-L26

And the API documentation.

Reply:
Good point.